UPGRADE.md

```diff
@@ -2,6 +2,25 @@
 
 ## dev-develop
 
+### System Collections
+
+Remove the config `sulu_contact.form.avatar_collection` and not it you will need it in the sql statement below for the
+placeholder `{old-avatar-collection}` (default value is `1`).
+
+Update the database schema and then update the data-fixtures by running following sql statement.
+
+```bash
+app/console doctrine:schema:update --force
+```
+
+```sql
+UPDATE me_collection_types SET collection_type_key='collection.default', name='Default' WHERE id=1;
+INSERT INTO me_collection_types (id, name, collection_type_key) VALUES ('2', 'System Collections', 'collection.system');
+UPDATE me_media SET idCollections={new-system-collection-id} WHERE idCollections={old-avatar-collection};
```

Comment:
This line should be executed after the site was reloaded, otherwise the collection does not exist yet.